prismic/custom-types/events.json

```diff
@@ -36,9 +36,16 @@
     },
     "timestamp" : {
       "type" : "Timestamp",
-      "fieldset" : "Date and time",
+      "fieldset" : "Start date and time",
       "config" : {
-        "label" : "Event time"
+        "label" : "Event start time"
+      }
+    },
+    "timestampEnd" : {
+      "type" : "Timestamp",
+      "fieldset" : "End date and time (for multi-day events)",
```

Comment:
Hmm I suppose we could populate single day events `timestampEnd` too for consistency